src/launcher/index.ts

```diff
@@ -154,9 +164,22 @@ async function main() {
       flowrWindow = await createFlowrWindow(store)
       FullScreenManager.applyDefaultActionOnWindow(flowrWindow)
       applicationManager.flowrWindow = flowrWindow
+
       flowrWindow.on('close', () => {
         flowrWindow = null
       })
+
+      flowrWindow.webContents.on('new-window', async (event, url) => {
+        event.preventDefault()
+
+        await openBrowserWindow(store, event, {
+          clearBrowsingDataAtClose: false,
+          openUrl: url,
+          maxTab : 1,
```

Comment:
Let's set it to 0 instead, to limit further navigation
```suggestion
          maxTab : 0,
```